proposals/lip-define_state_transitions_of_reward_module.md

```diff
@@ -0,0 +1,146 @@
+```
+LIP: <LIP number>
+Title:  Define state transitions of reward module
+Authors: Iker Alustiza <iker@lightcurve.io>
+         Mehmet Egemen Albayrak <mehmet.albayrak@lightcurve.io>
+Discussions-To: https://research.lisk.com/t/define-state-transitions-of-reward-module
+Type: Standards Track
+Created: <YYYY-MM-DD>
+Updated: <YYYY-MM-DD>
+Requires: Random Module LIP, BFT module LIP, and Token Module LIP.
+```
+
+## Abstract
+
+The reward module provides the base reward system for a blockchain developed with the Lisk SDK. 
+In this LIP, we specify the protocol logic that this module injects during the block lifecycle as well as the functions that can be called from off-chain services.
+
+## Copyright
+
+This LIP is licensed under the [Creative Commons Zero 1.0 Universal][creative].
+
+## Motivation
+
+The goal of this LIP is to modularize the rewards assignment logic when a new block is generated.
+For this purpose, we specify the reward module, which will make it easier to customize this logic and replace it if necessary. 
+
+## Rationale
+
+The reward module induces specific logic after the application of a block:
+
+* After applying the block, a certain quantity of the native token of the blockchain is minted and assigned to the block generator. The exact amount assigned to the block generator, i.e., the reward, is calculated depending of the rules of the [random module][randomLIP] and the [BFT module][BFTLIP].
+
+## Specification
+
+In this section, we specify the protocol logic in the lifecycle of a block injected by the reward module as well as the functions that can be called from off-chain services.
+This module does not contain commands, and has an empty state.
+It depends on the token, BFT, and random modules.
+
+### Constants
+
```

Comment:
You don't have the description column in the table

Reply:
is it always a must? specially for such small self explanatory table.